tests/zdd_app_blue_marathon1.5.json

```diff
@@ -0,0 +1,178 @@
+{
+  "apps": [
+    {
+      "id": "/nginx-green",
```

Comment:
This is the marathon 1.5 version of `zdd_app_blue.json` which also contains an app called `/nginx-green` - I am assuming that the original author did this on purpose, although I'm not sure why.